bin/get_pr_changed_files_list.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Use Github REST API to get list of changed files from most recent Pull request
+
+#
+#  export GITHUB_PATH=[token]
+#  export GITHUB_ORG=[orgname]
+# github_rest_get_pr_changed_files_list.sh [repo_name] [output_filename]
```

Comment:
worth deleting these comments?

Reply:
Those comments are usage instructions about how to run the script.